test/intl402/DurationFormat/prototype/formatToParts/formatToParts-styles-en.js

```diff
@@ -0,0 +1,125 @@
+    { type: "literal", value: " ", unit: "millisecond" },
+    { type: "unit", value: "msec", unit: "millisecond" },
+    { type: "literal", value: ", " },
+    { type: "integer", value: "456", unit: "microsecond" },
+    { type: "literal", value: " ", unit: "microsecond" },
+    { type: "unit", value: "μsec", unit: "microsecond" },
+    { type: "literal", value: " and " },
+    { type: "integer", value: "789", unit: "nanosecond" },
+    { type: "literal", value: " ", unit: "nanosecond" },
+    { type: "unit", value: "nsec", unit: "nanosecond" },
+  ],
+  long: [
+    { type: "integer", value: "7", unit: "hour" },
+    { type: "literal", value: " ", unit: "hour" },
+    { type: "unit", value: "hours", unit: "hour" },
+    { type: "literal", value: ", " },
+    { type: "integer", value: "8", unit: "minute" },
+    { type: "literal", value: " ", unit: "minute" },
+    { type: "unit", value: "minutes", unit: "minute" },
+    { type: "literal", value: ", " },
+    { type: "integer", value: "9", unit: "second" },
+    { type: "literal", value: " ", unit: "second" },
+    { type: "unit", value: "seconds", unit: "second" },
+    { type: "literal", value: ", " },
+    { type: "integer", value: "123", unit: "millisecond" },
+    { type: "literal", value: " ", unit: "millisecond" },
+    { type: "unit", value: "milliseconds", unit: "millisecond" },
+    { type: "literal", value: ", " },
+    { type: "integer", value: "456", unit: "microsecond" },
+    { type: "literal", value: " ", unit: "microsecond" },
+    { type: "unit", value: "microseconds", unit: "microsecond" },
+    { type: "literal", value: " and " },
+    { type: "integer", value: "789", unit: "nanosecond" },
+    { type: "literal", value: " ", unit: "nanosecond" },
+    { type: "unit", value: "nanoseconds", unit: "nanosecond" },
+  ],
+  narrow: [
+    { type: "integer", value: "7", unit: "hour" },
+    { type: "unit", value: "h", unit: "hour" },
+    { type: "literal", value: " " },
+    { type: "integer", value: "8", unit: "minute" },
+    { type: "unit", value: "m", unit: "minute" },
+    { type: "literal", value: " " },
+    { type: "integer", value: "9", unit: "second" },
+    { type: "unit", value: "s", unit: "second" },
+    { type: "literal", value: " " },
+    { type: "integer", value: "123", unit: "millisecond" },
+    { type: "unit", value: "ms", unit: "millisecond" },
+    { type: "literal", value: " " },
+    { type: "integer", value: "456", unit: "microsecond" },
+    { type: "unit", value: "μs", unit: "microsecond" },
+    { type: "literal", value: " " },
+    { type: "integer", value: "789", unit: "nanosecond" },
+    { type: "unit", value: "ns", unit: "nanosecond" },
+  ],
+  digital: [
+    { type: "integer", value: "7", unit: "hour" },
+    { type: "literal", value: ":", unit: "hour" },
+    { type: "integer", value: "08", unit: "minute" },
+    { type: "literal", value: ":", unit: "minute" },
+    { type: "integer", value: "09", unit: "second" },
```

Comment:
I think you missed the `.123456789`

Reply:
Hi, I have a doubt if by default we do
```
8. If listStyle is "digital", then
      a. Set listStyle to "short".
```

To see the sub-second values we shouldn't define the number of fractionalDigits ?       

```js

new Intl.DurationFormat('en',{ style : 'digital' , fractionalDigits: 9 }).formatToParts( {
  hours: 7,
  minutes: 8,
  seconds: 9,
  milliseconds: 123,
  microseconds: 456,
  nanoseconds: 789,
});

// Output  something like 

    { type: "integer", value: "7", unit: "hour" },
    { type: "literal", value: ":", unit: "hour" },
    { type: "integer", value: "08", unit: "minute" },
    { type: "literal", value: ":", unit: "minute" },
    { type: "integer", value: "09", unit: "second" },
    { type: "literal", value: ".", unit: "second" },
    { type: "integer", value: "123456789", unit: "millisecond" },


```

Reply:
I think this is done in [PartitionDurationFormatPattern](https://tc39.es/proposal-intl-duration-format/#sec-partitiondurationformatpattern) step 4.j.iv.

The default `fractionalDigits` is 0, so I think this test is correct.

Reply:
You're right! Sorry for the false alarm.